Create DIVD-2024-00033.md (#816)
DIVD-NL/web-csirt: _cases/2024/DIVD-2024-00033.md

---
layout: case
title: "ServiceNow - unauthenticated remote code execution (RCE)"
author: Alwin Warringa
lead: Alwin Warringa
excerpt: "Multiple vulnerabilities have been found in ServiceNow. Combining these vulnerabilities could enable an unauthenticated user to remotely execute code within the context of the Now Platform"
researchers:
- Alwin Warringa
- Stan Plasmeijer
- Victor Plasman
cves:
- CVE-2024-4879
- CVE-2024-5178
- CVE-2024-5217
product:
- ServiceNow
versions:
- Release Utah, below patch 10 Hot Fix 3
- Release Utah, below patch 10a Hot Fix 2
- Release Vancouver, below patch 6 Hot Fix 2
- Release Vancouver, below patch 7 Hot Fix 3b
- Release Vancouver, below patch 8 Hot Fix 4
- Release Vancouver, below patch 9
- Release Vancouver, below patch 10
- Release Washington, below patch 1 Hot Fix 2b
- Release Washington, below patch 2 Hot Fix 2
- Release Washington, below patch 3 Hot Fix 1
- Release Washington, below patch 4

> recommendation: "Check for the patched versions and get those installed"

Reviewer: ```suggestion
recommendation: "Install a patched version of SericeNow"
```

> workaround: "No"

Reviewer: ```suggestion
workaround: "No workaround available"
```

patch_status: Released
status : Open
start: 2024-07-13
end:
timeline:
- start: 2024-07-13
  end:
  event: "DIVD starts researching the vulnerability."
- start: 2024-07-13
  end:
  event: "DIVD finds fingerprint, preparing to scan."
- start: 2024-07-17
  end:
  event: "DIVD starts scanning the internet for vulnerable instances."
- start: 2024-07-17
  end:
  event: "DIVD starts notifying network owners with a vulnerable instance in their network."
# ips: 0

---

## Summary

> ServiceNow has addressed an input validation vulnerability that was identified in Vancouver and Washington DC ServiceNow Platform releases. This vulnerability could enable an unauthenticated user to remotely execute code within the context of the Now Platform. ServiceNow applied an update to hosted instances, and ServiceNow released the update to our partners and self-hosted customers.

Reviewer: Ik zie in de effected version ook Utah staan, maar deze staat niet in dee paragraaf.

> ServiceNow has addressed an input validation vulnerability that was identified in Vancouver and Washington DC ServiceNow Platform releases. This vulnerability could enable an unauthenticated user to remotely execute code within the context of the Now Platform. ServiceNow applied an update to hosted instances, and ServiceNow released the update to our partners and self-hosted customers.

Reviewer: ```suggestion
ServiceNow has addressed an input validation vulnerability that was identified in Vancouver and Washington DC ServiceNow Platform releases. This vulnerability could enable an unauthenticated user to remotely execute code within the context of the Now Platform. ServiceNow applied an update to hosted instances, and ServiceNow released the update to their service partners and customers that are running a self-hosted version.
```
